tests/Tokens/setInterestRateModelTest.js

```diff
@@ -22,38 +22,35 @@ describe('CToken', function () {
     });
 
     it("fails if called by non-admin", async () => {
-      expect(
-        await send(cToken, 'harnessSetInterestRateModelFresh', [newModel._address], {from: accounts[0]})
-      ).toHaveTokenFailure('UNAUTHORIZED', 'SET_INTEREST_RATE_MODEL_OWNER_CHECK');
+      await expect(send(cToken, 'harnessSetInterestRateModelFresh', [newModel._address], {from: accounts[0]}))
```

Comment:
Codacy has a fix for the issue: [Replace `send(cToken,·'harnessSetInterestRateModelFresh',·[newModel._address],·{from:·accounts[0]}))` with `⏎········send(cToken,·"harnessSetInterestRateModelFresh",·[newModel._address],·{⏎··········from:·accounts[0],`](https://app.codacy.com/gh/cloudwalk/compound-protocol/pullRequest?prid=9803552)
```suggestion
      await expect(
```